### ISSUE TITLE
Necro Stalker and Venom Stalker mobs use their Poison Charge ability in melee range

### ISSUE THREAD

**Issue by balakethelock** [slowtorta/turtlewow-bug-tracker#3412]
**Please, provide a clear description what the bug is:**
Two mobs in Naxxramas have a charge ability that is incorrectly targeting melee players and placing poison on all the melee players.

**Steps to reproduce the behavior with as much detail as possible:**

1. Engage combat with Venom Stalkers in the beginning of spider wing, or Necro Stalkers in the outer circle of the same wing.
2. Stand in melee range
3. See that they targeted you for their poison charge ability 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=15976#abilities
NPC: https://database.turtle-wow.org/?npc=16453

Spell: https://database.turtle-wow.org/?spell=28431

**Expected behavior. Describe how it should work:**
This mob is supposed to work like the Lava Surger mob in molten core:
-It charges the furthest player in range.
-When everybody is within 10 yards range, it can't target anyone for the ability and thus doesn't use the poison charge ability anymore.

This is the blizzlike vanilla tactic for dealing with these mobs.

**Comment by MarcelineVQ** [comment]
If not matching blizzlike is an error here would be possible to fix by changing the `rangeIndex` of https://database.turtle-wow.org/?spell=28431 from `5` to `10`?
`rangeIndex` `5` gives the spell 0-40y range, `10` would give it 10-40y range and, if the code respects `rangeIndex`, eliminate the melee spam.

**Comment by balakethelock** [comment]
> If not matching blizzlike is an error here would be possible to fix by changing the `rangeIndex` of https://database.turtle-wow.org/?spell=28431 from `5` to `10`? `rangeIndex` `5` gives the spell 0-40y range, `10` would give it 10-40y range and, if the code respects `rangeIndex`, eliminate the melee spam.

Yes and no, the range of spells is written by blizzard in client files, the spell SHOULD correctly have 0-40yd range, even if blizzard coded the mob that uses the spell to never cast it in melee range.

Better fix is improving the targeting, lava surgers in MC use a similar spell that has 0-40yd range, but they don't use it in melee because they are coded to not use it.
